packages/cli/src/eventbus/auditor.service.ts

```diff
@@ -0,0 +1,349 @@
+import { EventEmitter } from 'node:events';
+import { Service } from 'typedi';
+import { MessageEventBus } from './MessageEventBus/MessageEventBus';
+import { Redactable } from '@/decorators/Redactable';
+import type { AuditEventArgs } from './audit.types';
+import type { IWorkflowBase } from 'n8n-workflow';
+
+/**
+ * Service for adding events from `EventSender` to the audit log.
+ */
+@Service()
+export class AuditorService extends EventEmitter {
```

Comment:
Since this is not yet used anywhere, is the idea in the future that this would subscribe to events on `EventSender`? I don't see a reason why this class needs to be an `EventEmitter`. We could instead in `setupHandlers` subscribe directly to `EventSender`:

```ts
this.eventSender.on("workflow-created`, event => this.workflowCreated(event))
```

I think we are a bit obsessed with naming everything `*Service`. We could rename this to `AuditEventRelay` or `AuditEventRouter` as it relays events from the `EventSender` to the `MessageEventBus`